lib/livebook/teams/requests.ex

```diff
@@ -85,6 +86,7 @@ defmodule Livebook.Teams.Requests do
     params = %{name: secret.name}
 
     delete("/api/v1/org/secrets", params, headers)
+    |> dispatch_messages(team)
```

Comment:
What if we do it automatically after every request? As soon as we verify it was a JSON response and we decode it?

Reply:
Does it make sense for Org requests? (new org/join org)

Reply:
Because It would only dispatch messages for requests that makes sense. WDYT?

Reply:
For requests where it doesn't make sense, the field won't  be there, so we just detect the case there is no field and do nothing. :)